packages/components/src/simple-select-control/index.js

```diff
@@ -87,19 +91,22 @@ class SimpleSelectControl extends Component {
 				contentClassName="woocommerce-simple-select-control__dropdown-content"
 				position="center"
 				renderToggle={ ( { isOpen, onToggle } ) => (
-					<Button
-						className="woocommerce-simple-select-control__selector"
-						onClick={ () => this.handleOnClick( onToggle ) }
-						onFocus={ () => this.handleOnFocus() }
-						aria-expanded={ isOpen }
-						aria-label={ ! isEmpty ? sprintf(
-							/* translators: Label: Current Value for a Select Dropddown */
-							__( '%s: %s' ), label, currentOption && currentOption.label
-						) : label }
-					>
-						<span className="woocommerce-simple-select-control__label">{ label }</span>
-						<span className="woocommerce-simple-select-control__value">{ currentOption && currentOption.label }</span>
-					</Button>
+					<Fragment>
+						<Button
+							className="woocommerce-simple-select-control__selector"
+							onClick={ () => this.handleOnClick( onToggle ) }
+							onFocus={ () => this.handleOnFocus() }
+							aria-expanded={ isOpen }
+							aria-label={ ! isEmpty ? sprintf(
+								/* translators: Label: Current Value for a Select Dropddown */
+								__( '%s: %s' ), label, currentOption && currentOption.label
+							) : label }
+						>
+							<span className="woocommerce-simple-select-control__label">{ label }</span>
+							<span className="woocommerce-simple-select-control__value">{ currentOption && currentOption.label }</span>
+						</Button>
+						{ !! help && <p id={ id + '__help' } className="components-base-control__help">{ help }</p> }
```

Comment:
Should this class name be `woocommerce-simple-select-control__help` instead?

Reply:
I went with `components-base-control__help` here and `components-base-control` in the `Dropdown` classes above, so that these inherit the same base WP component styles as other Muriel or WP core components. This matches the help text returned by other WP components.